<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnippetViewer</class>
 <widget class="QWidget" name="SnippetViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="txtPanel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="backNavButton">
            <property name="toolTip">
             <string>Backward</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fwdNavButton">
            <property name="toolTip">
             <string>Forward</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="homeButton">
            <property name="toolTip">
             <string>Return to initial page</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reloadButton">
            <property name="toolTip">
             <string>Reload</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="toolTip">
             <string>Stop loading</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QComboBox" name="urlEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="maxVisibleItems">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="navButton">
            <property name="toolTip">
             <string>Go by URL</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="barLoading">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="searchEdit">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="backButton">
            <property name="toolTip">
             <string>Search back</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fwdButton">
            <property name="toolTip">
             <string>Search forward</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="comboZoom">
          <property name="toolTip">
           <string>Zoom</string>
          </property>
          <property name="currentIndex">
           <number>5</number>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <item>
           <property name="text">
            <string>10%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>75%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>110%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>125%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>150%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>175%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>750%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000%</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="transButton">
          <property name="text">
           <string>Translate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWebView" name="txtBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtBrowser</tabstop>
  <tabstop>backNavButton</tabstop>
  <tabstop>fwdNavButton</tabstop>
  <tabstop>homeButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>searchEdit</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>fwdButton</tabstop>
  <tabstop>comboZoom</tabstop>
  <tabstop>transButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
